flink-tests/src/test/java/org/apache/flink/test/iterative/aggregators/AggregatorsITCase.java

```diff
@@ -117,7 +107,9 @@ public boolean filter(Long value) throws Exception {
 			}
 		}).withBroadcastSet(solution, "SOLUTION")).output(new DiscardingOutputFormat<Long>());
 		env.execute();
-		expected = testString; // this will be a useless verification now.
+		String expected = testString; // this will be a useless verification now.
+
+		compareResultsByLinesInMemory(expected, resultPath);
```

Comment:
this is unnecessary and can be removed as the actual check is done in the function.